src/MastodonConnectionHelper.cs

```diff
@@ -36,11 +37,31 @@ public static async Task<List<string>> GetPinnedTagsAsync()
         await conn.OpenAsync();
 
         var res = new List<string>();
-        await using var cmd = new NpgsqlCommand("SELECT DISTINCT col->'params'->>'id' FROM web_settings, json_array_elements(data->'columns') col WHERE col->>'id' = 'HASHTAG' AND col->'params'->>'id' IS NOT NULL ORDER BY col->'params'->>'id' ASC", conn);
+        // Column 0: the 'original' tag with was pinned
+        // Column 1: Config of 'Include additional tags for this column' this includes the the tags in 'any' array.
+        await using var cmd = new NpgsqlCommand("SELECT DISTINCT col->'params'->>'id', col->'params'->>'tags' FROM web_settings, json_array_elements(data->'columns') col WHERE col->>'id' = 'HASHTAG' AND col->'params'->>'id' IS NOT NULL ORDER BY col->'params'->>'id' ASC", conn);
         await using var reader = await cmd.ExecuteReaderAsync();
         while (await reader.ReadAsync())
+        {
             res.Add(reader.GetString(0));
+            if (! reader.IsDBNull(1))
+            {
+                JsonDocument doc = JsonDocument.Parse(reader.GetString(1));
 
+                JsonElement anyArray = doc.RootElement.GetProperty("any");
+                foreach (JsonElement item in anyArray.EnumerateArray())
+                {
+                    JsonElement value = item.GetProperty("value");
+                    if (value.ValueKind != JsonValueKind.Null)
+                    {
+                        String? valuestring = value.GetString();
+                        if (! valuestring.IsNullOrEmpty())
+                            res.Add(valuestring);
```

Comment:
Consider checking for duplicates tags here, now that we can't rely on `SELECT DISTINCT` to do that for us

Reply:
Ope, I commented too soon. I see now there is a `.Distinct()` after this function is executed. I am dismissing this commented!!

As an aside, thanks for these PRs, I am super excited for this functionality and how quickly it went from Issue to PR — cheers!!!